test/llvmpasses/Makefile

```
@@ -9,6 +9,7 @@ TESTS_jl := $(patsubst $(SRCDIR)/%,%,$(wildcard $(SRCDIR)/*.jl))
 TESTS := $(TESTS_ll) $(TESTS_jl)
 
 . $(TESTS):
+	$(MAKE) -C $(JULIAHOME)/deps install-llvm-tools
```

```suggestion
	$(MAKE) -C $(BUILDDIR)/deps install-llvm-tools
```
(I don't remember if we make the direct JULIAHOME work, but this should work)

---

`JULIAHOME` is defined 10 lines above: https://github.com/JuliaLang/julia/blob/b24341cc4156b81618d1fa269e71dd847abf9df0/test/llvmpasses/Makefile#L2